com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/PyFunctionHashRemover/PyFunctionHashRemover.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" ?>
+<operatorModel
```

Comment:
I assume you mean a Python Filter?  Not sure I get why this is allowed since the PyFunctionFilter xml of the output port says: <tupleMutationAllowed>false</tupleMutationAllowed>
Am I missing something?